_targets.R

```diff
@@ -5,45 +5,79 @@ library(tibble)
 options(tidyverse.quiet = TRUE)
 library(tidyverse)
 
-tar_option_set(packages = c("tidyverse", "dataRetrieval", "urbnmapr", "rnaturalearth", "cowplot", "lubridate"))
+tar_option_set(packages = c("tidyverse", "dataRetrieval", "urbnmapr",
+                            "rnaturalearth", "cowplot", "lubridate",
+                            "leaflet", "leafpop", "htmlwidgets"))
 
 # Load functions needed by targets below
 source("1_fetch/src/find_oldest_sites.R")
 source("1_fetch/src/get_site_data.R")
 source("2_process/src/tally_site_obs.R")
+source("2_process/src/summarize_targets.R")
 source("3_visualize/src/map_sites.R")
+source("3_visualize/src/plot_data_coverage.R")
 source("3_visualize/src/plot_site_data.R")
+source("3_visualize/src/map_timeseries.R")
 
 # Configuration
-states <- c('WI','MN','MI','IL','IN','IA')
+states <- c('WI','MN','MI','IL','IN','IA','OH')
 parameter <- c('00060')
 
+mapped_by_state_targets <- tar_map(
+  tibble(state_abb = states) %>%
+    mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
+  tar_target(nwis_inventory, dplyr::filter(oldest_active_sites,
+                                           state_cd == state_abb)),
+  tar_target(nwis_data, get_site_data(nwis_inventory, state_abb, parameter)),
+  # Insert step for tallying data here
+  tar_target(tally, tally_site_obs(nwis_data)),
+  # Insert step for plotting data here
+  tar_target(timeseries_png,
+             plot_site_data(state_plot_files,
+                            nwis_data,
+                            parameter),
+             format = 'file'),
+  names = state_abb
+)
```

Comment:
I think you may be missing an argument to `tar_map()` that allows you to do the downstream combining -- you should have `unlist = FALSE` here.